src/main/scala/com/campudus/tableaux/controller/TableauxController.scala

```diff
@@ -30,14 +30,14 @@ class TableauxController(override val config: TableauxConfig, override protected
       annotationType: CellAnnotationType,
       value: String
   ): Future[CellLevelAnnotation] = {
-    checkArguments(greaterZero(tableId), greaterZero(columnId), greaterZero(rowId))
+    checkArguments(greaterZero(tableId), greaterThan(columnId, -1, "columnId"), greaterZero(rowId))
```

Comment:
Would it make sense to add functions like `isValidColumnId` instead of `greaterThan(...)` ? (Same would apply for `greaterZero(...)`)

Reply:
Create a task for that.